codalab/lib/bundle_cli.py

```diff
@@ -1215,7 +1215,15 @@ def auto_convert_type(value):
     @Commands.command(
         'workers',
         help=['Display information about workers that you have connected to the CodaLab instance.'],
-        arguments=(),
+        arguments=(
+            Commands.Argument(
+                '-c',
```

Comment:
In other places, we use `-u` for printing only the uuid (for worksheets and bundles), so maybe use that for cinsistency?